flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/TimerSerializer.java

```diff
@@ -230,16 +264,29 @@ public TypeSerializerConfigSnapshot snapshotConfiguration() {
 	 */
 	public static class TimerSerializerConfigSnapshot<K, N> extends CompositeTypeSerializerConfigSnapshot {
 
+		private static final int VERSION = 0;
```

Comment:
I think the versioning scheme for the configs / serializers we tend to use starts from version 1. Should we stick with that?